crawler/processing/rabbit_message_processor.py

```diff
@@ -16,14 +15,18 @@ def __init__(self, schema_registry, basic_publisher, config):
         self._basic_publisher = basic_publisher
         self._config = config
 
-        self._processors = {
-            RABBITMQ_SUBJECT_CREATE_PLATE: CreatePlateProcessor(
-                self._schema_registry, self._basic_publisher, self._config
-            ),
-            RABBITMQ_SUBJECT_UPDATE_SAMPLE: UpdateSampleProcessor(
-                self._schema_registry, self._basic_publisher, self._config
-            ),
-        }
+        self._build_processors()
+
+    def _build_processors(self):
+        self._processors = {}
+        for subject in self._config.PROCESSORS.keys():
+            self._processors[subject] = self._build_processor_for_subject(subject)
```

Comment:
Probably a style choice, but is there any reason not to make this a lazy loaded property getter instead?

i.e. set `self.__processors` to `None` in the constructor, then implement this as:

```suggestion
    @property
    def _processors(self):
        if self.__processors is None:
            self.__processors = {subject: self._build_processor_for_subject(subject) for subject in self._config.PROCESSORS.keys()}
        
        return self.__processors
```

Reply:
yes looks good! I'll add it